kong/cmd/restart.lua

```diff
@@ -49,6 +49,8 @@ Options:
  --run-migrations (optional boolean)  optionally run migrations on the DB
  --db-timeout     (default 60)
  --lock-timeout   (default 60)
+ --conf-template-flags     (optional string) flags that can be used to control
+                                             how Nginx configuration templates are rendered
```

Comment:
Similarly here perhaps. Move add `--nginx-conf-flags` just below the ``--nginx-conf` and align the `(optional` etc.